server/src/main/java/com/tencent/rss/server/MultiStorageManager.java

```diff
@@ -140,6 +141,16 @@ public MultiStorageManager(ShuffleServerConf conf, String shuffleServerId) {
       throw new IllegalArgumentException("The value of maxShuffleSize must be positive");
     }
 
+    long pendingEventTimeoutS = conf.get(ShuffleServerConf.PENDING_EVENT_TIMEOUT_SEC);
+    if (pendingEventTimeoutS <= 0) {
+      throw new IllegalArgumentException("The value of pendingEventTimeoutS must be positive");
+    }
+
+    double maxShuffleForceUploadTimeRatio = conf.get(ShuffleServerConf.SHUFFLE_MAX_FORCE_UPLOAD_TIME_RATIO);
```

Comment:
typo ? ratio -> radio?
radio must between 0 and 100?

Reply:
It's ratio, radio is a electronic term. Add range check and log according to your suggestion.